ui/page/overview/wallet_status.go

```diff
@@ -179,7 +179,14 @@ func (pg *AppOverviewPage) syncDormantContent(gtx layout.Context, uniform layout
 			}),
 			layout.Rigid(func(gtx C) D {
 				if pg.WL.MultiWallet.IsSynced() {
-					return pg.connectionPeer(gtx)
+					return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
+						layout.Rigid(func(gtx C) D {
+							return pg.connectionPeer(gtx)
+						}),
+						layout.Rigid(func(gtx C) D {
+							return pg.autoSyncSec(gtx)
+						}),
+					)
```

Comment:
Do we only want to show the autoSync switch _after_ sync completes?

Reply:
Yes